lib/dialyzer/doc/src/dialyzer.xml

```diff
@@ -220,6 +232,13 @@ dialyzer --apps inets ssl ./ebin ../other_lib/ebin/my_module.beam</code>
         <p>When using Dialyzer from the command line, send the analysis
           results to the specified outfile rather than to <c>stdout</c>.</p>
       </item>
+      <tag><c>--metrics_file file</c></tag>
+      <item>
+        <p> Write metrics about Dialyzer's incrementality (e.g. total number of
```

Comment:
We generally avoid using abbreviations such as "e.g." and "i.e." in the documentation:

```suggestion
        <p> Write metrics about Dialyzer's incrementality (for example, total number of
```